orca-retrofit/orca-retrofit.gradle

```diff
@@ -19,14 +19,14 @@ apply from: "$rootDir/gradle/groovy.gradle"
 dependencies {
   api("com.squareup.retrofit:retrofit")
   api("com.squareup.retrofit:converter-jackson")
+  api("com.jakewharton.retrofit:retrofit1-okhttp3-client:1.1.0")
```

Comment:
orca-retrofit is depended on by a bunch of modules so I'd rather avoid adding things to its api classpath

You could add this version declaration to the orca-bom:
https://github.com/spinnaker/orca/blob/d123a1480c4922fa56734254c653c2b6b763a215/orca-bom/orca-bom.gradle#L41

keep  `implementation("com.jakewharton.retrofit:retrofit1-okhttp3-client")`  in here as an implementation dependency here, and in any extension module add an implementation dependency there as well